Alignment/HIPAlignmentAlgorithm/src/HIPAlignmentAlgorithm.cc

```diff
@@ -392,8 +392,8 @@ void HIPAlignmentAlgorithm::terminate(const edm::EventSetup& iSetup)
 
     bool test = calcParameters(ali,SetScanDet.at(0),SetScanDet.at(1),SetScanDet.at(2));
     // if successful, apply parameters
-    if (test) { 
-      edm::LogWarning("Alignment") << "now apply params";
+    if (test){
+      //edm::LogWarning("HIPAlignmentAlgorithm") << "now apply params";
```

Comment:
The log file becomes huge if the if statement passes, and it is not really an instructive check anyway.